src/main/java/frc/robot/subsystems/swerve/MissileShellSwerveSubsystem.java

```diff
@@ -51,11 +56,15 @@ public MissileShellSwerveSubsystem() {
         steerMotor = MotorUtil.createSparkMax(1);
         steerMotor.setIdleMode(IdleMode.kBrake);
 
-        steerEncoder = steerMotor.getAnalog(SparkMaxAnalogSensor.Mode.kAbsolute);
-        steerEncoder.setPositionConversionFactor(2 * Math.PI / 3.3); // 3.3V -> 2pi
+        steerAbsoluteEncoder = steerMotor.getAnalog(SparkMaxAnalogSensor.Mode.kAbsolute);
+        steerAbsoluteEncoder.setPositionConversionFactor(STEER_VOLTS_TO_RADIANS);
+
+        steerEncoder = steerMotor.getEncoder();
+        steerEncoder.setPositionConversionFactor(STEER_ROTATIONS_TO_RADIANS);
+        steerEncoder.setPosition(steerAbsoluteEncoder.getPosition()); // Set initial position to absolute value
 
         steerPidController = steerMotor.getPIDController();
-        steerPidController.setFeedbackDevice(steerEncoder);
+        //steerPidController.setFeedbackDevice(steerAbsoluteEncoder);
```

Comment:
does this not need to be called on the built-in encoder

Reply:
It defaults to the built-in relative encoder. We actually didn't use this method at all last year; I only discovered it while researching analog feedback.